.github/workflows/publish.yml

```diff
@@ -33,7 +33,6 @@ jobs:
         with: 
           args: publish
         env:
-          ORG_GRADLE_PROJECT_publishUser: ${{ secrets.BINTRAY_USER }}
-          ORG_GRADLE_PROJECT_publishApiKey: ${{ secrets.BINTRAY_API_KEY }}
-
-
+          ORG_GRADLE_PROJECT_artifactory_contextUrl: ${{ secrets.ARTIFACTORY_CONTEXT_URL }}
```

Comment:
Have these secrets been added?

Reply:
Hah, I was so slow with my review that the same comment came in midway - resolving.